docs/resiliency/index.asciidoc

```diff
@@ -204,6 +204,13 @@ Commonly used filters are cached in Elasticsearch. That cache is limited in size
 
 While we are working on a longer term solution ({GIT}9176[#9176]), we introduced a minimum weight of 1k for each cache entry. This puts an effective limit on the number of entries in the cache. See {GIT}8304[#8304] (STATUS: DONE, fixed in v1.4.0)
 
+[float]
+=== Do not allow stale shards to automatically be promoted to primary (STATUS: ONGOING)
```

Comment:
"Do not allow stale shards to automatically be promoted to primary if all other copies are lost" ?